### ISSUE TITLE
Corner points of extend attribute influence text direction after rotation

### ISSUE THREAD

**Issue by abuntrock** [OpenModelica/OpenModelica#11120]
### Description
The position of the extend attribute points influence the text direction after the text object is rotated.

I am not sure if this is a duplicate of #10650 but there the rectangle and the text objects had the same extent and origin values indicating a difference between the handling of rectangle and text objects. 

### Steps to Reproduce

1. Consider the following model. The first text object has its extend points at the bottom left and top right. The second text object has those points at the top left and bottom right.
```modelica
model Textrotation
  annotation(
    Icon(graphics = {
    Text(origin = {-134, 48},extent = {{54, 52}, {213, 72}}, textString = "Some Text A"), 
    Text(origin = {560, -80}, extent = {{-640, 160}, {-480, 140}}, textString = "Some Text B"), 
    Rectangle(origin = {-1, -7}, fillColor = {28, 108, 200}, fillPattern = FillPattern.Solid, extent = {{-63, 39}, {63, -39}})}));
end Textrotation;
```
2. Instantiate this model three times inside a new class. Rotate one instance 90° in clockwise direction and a second one 90° in anti clockwise direction. For the rotated instances the two textboxes must be read in different directions as shown in the attached screenshot.

### Expected Behavior

The text direction is the same after rotation independent of the relationship between the points of the extent statement.

### Screenshots

My collegue @niklwors will add the screenshots as a comment.

### Version and OS

 - OpenModelica Version: v1.22.0-dev-278-ge7b7ec3ee1 (64-bit)
 - OS: Windows 10, 64 bit

Thanks for considering this
Aaron Buntrock - Bosch Rexroth

**Comment by abuntrock** [comment]
This issue is fixed with the current nightly (v1.23.0-dev-257-gcfdfeb87e5). Most probably by #11691.
Thanks a lot

**Comment by abuntrock** [comment]
I forgot to close it with the previous comment
